src/package/compressed.rs

```diff
@@ -186,11 +222,11 @@ impl Package {
             .into_diagnostic()
             .wrap_err(DeserializationError(ManagedFile::Manifest))?;
 
-        let manifest = String::from_utf8(manifest)
+        let manifest_str = String::from_utf8(manifest)
```

Comment:
If you disagree with the original way of reassignment I would suggest nesting a block for decoding so you drop the str variant as soon as the block ends

Reply:
f77d2faf2d04a4765aace473cea58f2ed5a09cd0